openedx/features/partners/views.py

```diff
@@ -47,8 +46,10 @@
 def dashboard(request, slug):
     partner = get_object_or_404(Partner, slug=slug)
     courses = get_partner_recommended_courses(partner.slug, request.user)
+    registration_countries = get_registration_countries()
     return render_to_response('features/partners/dashboard.html', {'recommended_courses': courses,
-                                                                   'slug': partner.slug, 'partner': partner})
+                                                                   'slug': partner.slug, 'partner': partner,
+                                                                   'registration_countries': registration_countries, })
```

Comment:
It would be great if we can simplify this with following code
```
    context = {
        'recommended_courses': courses,
        'slug': partner.slug, 'partner': partner,
        'registration_countries': get_registration_countries(),
    }
    return render_to_response('features/partners/dashboard.html', context)
```